bin/sign.py

```diff
@@ -226,6 +229,36 @@ def json_tempfile(path: Path, data: JSON) -> Generator[None, None, None]:
     path.unlink()
 
 
+def parse_env_vars(runtime: str) -> tuple[str, str, str, JSON]:
+    """Parse and return environment variables"""

```

Comment:
Is `"""` required in Python to escape '`"`. I ask because it's `""` in C#. :)

Reply:
This is a standard python docstring convention. https://peps.python.org/pep-0257/#one-line-docstrings

Reply:
Technically, it could just be a `"`. You can see this if you do
```python
>>> def foobar():
...     "Amazingly, this is a valid docstring."
...     pass
...    
>>> foobar.__doc__
'Amazingly, this is a valid docstring.
```
, but it's very much against convention to do that and you almost always see docstrings with `"""`. The documentation that @mvanchaa linked points out the main reason
> Triple quotes are used even though the string fits on one line. This makes it easy to later expand it.